bin/codemods/run.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const child_process = require( 'child_process' );
+const glob = require( 'glob' );
+
+/**
+ * Internal dependencies
+ */
+const config = require( './src/config' );
+
+function getLocalCodemodFileNames() {
+	// Returns all JS files in bin/codemods/src folder, except for config and helpers.
+	return glob.sync( 'bin/codemods/src/*.js' )
+		.filter( name => name !== 'config.js' ) // exclude utility files
+		.map( name => name.replace( /\.js$/, '' ) ); // strip extension from filename
+}
+
+function getValidCodemodNames() {
+	return [
+		...getLocalCodemodFileNames(),
+		...Object.getOwnPropertyNames( config.codemodArgs ),
+	].map( name => '- ' + name ).sort();
```

Comment:
with our filenames this may not matter, but check out `localeCompare()` for a good default string sorter!